src/components/feedback/Dialog/styles.less

```diff
@@ -0,0 +1,34 @@
+@import '~styles/mixins.less';
+
+.container {
+    padding: .grid-unit(2px)[] 0 .grid-unit(1px)[] 0;
+    border: 1px solid var(--color-black-alt4);
+    border-radius: 4px;
+    max-width: .grid-unit(35px)[];
```

Comment:
Should the width and height be hardcoded instead of just max-width? Perhaps a fixed width and flexible height (min- and max-height) and then resort to scrollable when the content overflows the max-height?